lib/provider/ui/windows/control-panel.tsx

```diff
@@ -0,0 +1,71 @@
+import type { ReactNode } from "react";
+import type { SelectOption } from "react95/dist/Select/Select.types";
+import type { Theme } from "react95/dist/types";
+
+import type { ThemeType } from "~/lib/atom";
+
+import { useAtom } from "jotai";
+import { useEffect, useState } from "react";
+import { GroupBox, Select } from "react95";
+
+import { themeAtom } from "~/lib/atom";
+import { Window } from "~/lib/window";
+
+const themeOptions: SelectOption<ThemeType>[] = [
+    { label: "Default", value: "original" },
+    { label: "Black and white", value: "black&white" },
+    { label: "Modern Dark", value: "modernDark" }
+];
+
+export function ControlPanel(): ReactNode {
+    const [theme, setTheme] = useAtom(themeAtom);
+    const [selectedTheme, setSelectedTheme] = useState<ThemeType>("original");
+
+    useEffect(() => {
+        setSelectedTheme(theme.type);
+    }, [theme.type]);
+
+    const handleThemeChange = async (selectedOption: SelectOption<ThemeType>): Promise<void> => {
+        const themes = await import("react95/dist/themes");
+
+        let newTheme: Theme;
+        switch (selectedOption.value) {
+            case "original": {
+                newTheme = themes.default.original;
+                break;
+            }
+            case "black&white": {
+                newTheme = themes.default.blackAndWhite;
+                break;
+            }
+            case "modernDark": {
+                newTheme = themes.default.modernDark;
+                break;
+            }
+            default: {
+                newTheme = themes.default.original;
+            }
+        }
+        setTheme({ type: selectedOption.value, value: newTheme });
+        setSelectedTheme(selectedOption.value);
+    };
```

Comment:
There is no need to use switch cases here, we can just use the keys & values of `await import("react95/dist/themes")` directly.